### PR TITLE
add getnighthawk-brand kit to brands section

### LINKED ISSUE #1780: [Brand] Create new section for GetNighthawk
**Current Behavior**
GetNighthawk is an upcoming Layer5 project. It now has a logo and small brand kit that needs to be made available on https://layer5.io/brand

**Desired Behavior**
On https://layer5.io/brand:
1. A new section for GetNighthawk needs to be created.
1. The zipped brand kit needs to be made available for download. 
[getnighthawk-brand-kit.zip](https://github.com/layer5io/layer5/files/6305014/getnighthawk-brand-kit.zip)
(updated)

1. The project logo should be displayed on the page.

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Thea10**
Signed-off-by: Cynthia <olucynakin@gmail.com>

**Description**

This PR fixes #1780

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit bf9a34e4c506d1d9000c3673a086e121363d4ee5 at: https://607861618a67fd49c48705dc--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit bf9a34e4c506d1d9000c3673a086e121363d4ee5 at: https://607861e9bc1dfd17c4331e1d--layer5ng.netlify.app

File: src/sections/Company/Brand/Brand-components/getnighthawk.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            </Button>
+          </a>
+        </Col>
+        <Col>
+          <p>
+            The GetNightHawk mark includes the GetNightHawk name & logo and any
+            word, phrase, image, or other designation that identifies the source
+            or origin of any Layer5 projects. PLease don't modify the marks or
+            use them in a confusing way, including suggesting sponsorship or
+            endorsement by Layer5 with another brand(including your own)
```
Comment (leecalcote) [inline src/sections/Company/Brand/Brand-components/getnighthawk.js:32]:
```suggestion
            endorsement by Layer5 with another brand (including your own).
```

**Review by leecalcote — APPROVED** [review]
Looks great. 👍 Good one, @Thea10

**Pull request merged**